### DISCUSSION TITLE
Sorting Column

### DISCUSSION THREAD

**Discussion by aloreaux** [Budibase/budibase#6626] [Help] [answered]
How do I go about making a unique sequential number sort in a table view (it is my first column). When I click on it all rows disappear. What am I doing wrong?

**Comment by melohagan** [comment, marked as answer]
Hi @aloreaux 

Budibase internal tables have an auto incrementing ID, known as the `Auto ID`.

Click **Auto columns**
![Screenshot 2022-07-15 at 17 02 11](https://user-images.githubusercontent.com/101575380/179262144-3e8b87e8-921b-4447-bc3b-cf70df6042df.png)

Then you should be able to sort it by clicking the header:
![Screenshot 2022-07-15 at 17 02 32](https://user-images.githubusercontent.com/101575380/179262215-4c769092-3397-4a31-91d0-673d407db874.png)
